From 253973e569a8536bc125a7764c12691a6cd2c08e Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Wed, 11 Feb 2026 20:23:18 +0100
Subject: [PATCH] arm64: dts: allwinner: sun50i-a64-pinephone: Add missing
 ramp-delay to DCDC2 regulator used for CPUX

Ramp delay description is needed for use during DVM during CPU clock
changes.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index d502c1392b8e..64a37fff2ca9 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -879,6 +879,7 @@ &reg_dcdc2 {
 	regulator-always-on;
 	regulator-min-microvolt = <1000000>;
 	regulator-max-microvolt = <1300000>;
+	regulator-ramp-delay = <5000>; /* 2.5mV / us (intentionally doubled) */
 	regulator-name = "vdd-cpux";
 };
 
-- 
2.43.0

